lib/thesis/view.ex

```diff
@@ -93,21 +93,29 @@ defmodule Thesis.View do
 
   Doctests:
 
-      iex> {:safe, editor} = Thesis.View.thesis_editor(%Plug.Conn{assigns: %{editable: true}})
+      iex> {:safe, editor} = Thesis.View.thesis_editor(%Plug.Conn{assigns: %{thesis_editable: true}})
       iex> String.contains?(editor, "/thesis/thesis.css")
       true
       iex> String.contains?(editor, "thesis-container")
       true
       iex> String.contains?(editor, "<script>")
       true
 
-      iex> Thesis.View.thesis_editor(%Plug.Conn{assigns: %{editable: false}})
+      iex> Thesis.View.thesis_editor(%Plug.Conn{assigns: %{thesis_editable: false}})
       {:safe, ""}
   """
   @spec thesis_editor(Plug.Conn.t) :: {:safe, String.t}
   def thesis_editor(conn) do
     if editable?(conn) do
-      editor = content_tag(:div, "", id: "thesis-container", data_ospry_public_key: ospry_public_key)
+      page = conn.assigns[:thesis_page]
+      redirect_url = page && page.redirect_url
+      template = page && page.template
+      templates = Enum.join(dynamic_templates, ",")
+      editor = content_tag(:div, "", id: "thesis-container",
+        data_ospry_public_key: ospry_public_key,
+        data_redirect_url: redirect_url,
+        data_template: template,
+        data_templates: templates)
```

Comment:
I'm adding a bunch of HTML dom attributes to the container to pass them through to Thesis's JS runtime.